services/agent/VERSION

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
Ouau! This is really a small version :-)))).  we normally start in 0.1.0 ... but OK! :-)))